.env.example

```diff
@@ -0,0 +1,22 @@
+### Required variables ###
+NEXT_PUBLIC_INFURA_TOKEN=
+
+### Optional variables ###
+# These variables are required only if you require a certain feature in the interface (e.g. Portis wallet)
+# Or overwrite the fallback values (set a different WalletConnect bridge)
+
+# Flag to switch to the production environment (redirect urls, gateway url, etc)
+NEXT_PUBLIC_IS_PRODUCTION=false
+
+# Latest supported safe version, used for upgrade prompts
+NEXT_PUBLIC_SAFE_VERSION=1.3.0
+
+# Access keys
+NEXT_PUBLIC_SENTRY_DSN=
+NEXT_PUBLIC_BEAMER_ID=
+
+# Wallet specific variables
+NEXT_PUBLIC_WC_BRIDGE=https://safe-walletconnect.gnosis.io/
```

Comment:
Need this also be empty?

Reply:
I left it there to let the dev know that it's the fallback/default value